packages/asc-ui/src/components/Header/HeaderLogoText.tsx

```diff
@@ -1,28 +1,33 @@
-import React from 'react'
+import React, {
+  AnchorHTMLAttributes,
+  FunctionComponent,
+  HTMLAttributes,
+} from 'react'
 import HeaderLogoTextStyle from './HeaderLogoTextStyle'
 import HeaderTitle from './HeaderTitle'
 
-export interface LogoProps
-  extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+export interface LogoProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
   href: string
   tall?: boolean
   title?: string
 }
 
-interface HeaderLogoTextProps extends React.HTMLAttributes<HTMLHeadingElement> {
+interface HeaderLogoTextProps extends HTMLAttributes<HTMLHeadingElement> {
+  as?: string | React.ComponentType
```

Comment:
Can you import `ComponentType` directly from React so that we do not have to use the `React` namespace. I was thinking perhaps we could use `ElementType` instead? I believe it actually accepts only known elements and also `ComponentType`.

Reply:
Done